### PR TITLE
For csv helper

### PR DESCRIPTION
**Pull request by YoungKwonJo**
I found a strange bug in "CatAnalyzer/src/BTagWeightEvaluator.cc".
It was compiled. however for based on root-files, it is getting wrong event weight value.

So I would like to use the old-one(my used) as CSVHelper.
CSVHelper gives a correct value. And, the following link is original code. I would like to keep the coding style. because it is not easy to debugging.
https://github.com/cms-ttH/MiniAOD/blob/master/MiniAODHelper/interface/CSVHelper.h

### CODE REVIEW

**Comment by YoungKwonJo** [comment]
And,now, the SF is not same between csv-file and root-files. so we need to see more.
btagsf_rootvscsvfile

**Comment by jhgoh** [comment]
Difference between csv file and root file is documented in the twiki page, but I don't know how it differs.

**Comment by jhgoh** [comment]
BTW, can you summarize what is the issue of wrong weighting factors?

**Comment by jhgoh** [comment]
Actually, the CSV/root file part of the BTagWeightEvaluator is just copy and paste of CSVHelper. What is the problem?

**Comment by YoungKwonJo** [comment]
no, CSVHelper is not using enum. perhaps, we have to see more hardly for BTagWeightEvaluator. 
And, the event weight is normally almost 1. however BTagWeightEvaluator with option root-file gave big number. so we need to debug.

**Comment by YoungKwonJo** [comment]
I guess we need to discuss by offline. I guess the effect of the difference is not big for our result and shape. however they have a little different.
I have checked the result(analysis) by using both as root-file and csv-file.
